<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>259</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>发送窗口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>530</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>接收窗口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_com">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>网络设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dis">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton_send</zorder>
  <zorder>textEdit_send</zorder>
  <zorder>textEdit_com</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_ip</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_port</zorder>
  <zorder>pushButton_dis</zorder>
  <zorder>pushButton_connect</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
